packages/vision/package.json

```diff
@@ -70,10 +70,40 @@
     "multiline": "^1.0.2",
     "node-uuid": "^1.4.7",
     "normalize-newline": "^2.0.0",
+    "nyc": "^10.3.0",
+    "power-assert": "^1.4.2",
     "proxyquire": "^1.7.10",
+    "sinon": "^2.2.0",
     "tmp": "^0.0.31"
   },
+  "nyc": {
```

Comment:
Maybe this can go into a .nycrc in the root directory and have the same effect. Or possibly better, we have a npm run-script, that is a small wrapper around the current `cover` task, that takes the module to test as an argument.